app/models/post.rb

```diff
@@ -0,0 +1,18 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :likes
+  has_many :comments
+
+  after_save :update_posts_counter
+
+  def recent_comments
+    Comment.where(['post_id = :id', { id: id.to_s }]).order(created_at: :desc).limit(5)
+  end
+
+  private
+
+  def update_posts_counter
```

Comment:
- [ ] [OPTIONAL] Active record has `increment` callback which increase integer underlying columns, you can take a look here:
```
def update_post_counter
  author.increment!(:posts_counter)
end
```